src/grimp/domain/valueobjects.py

```diff
@@ -86,21 +86,21 @@ def __hash__(self) -> int:
         return hash((str(self), self.line_contents))
 
 
-class Level(ValueObject):
+class Layer(ValueObject):
     """
-    A level within a layered architecture.
+    A layer within a layered architecture.
 
-    If level.independent is True then the layers within the level are considered
+    If layer.independent is True then the modules within the layer are considered
     independent. This is the default.
     """
 
     def __init__(
         self,
-        *layers: str,
+        *module_tails: str,
```

Comment:
🤔 Personally I think I still slightly prefer to call this simply `modules` or `module_names`. I know it's not _technically_ correct when `containers` are used, but I think overall it's probably still a bit simpler to understand. Not a strong opinion though.

Reply:
Let's stick with module tails. I think it's valuable to have precise language in the guts of Grimp. I don't think users of Grimp will have to engage much with this terminology in practice.